XCUITests/UnitTest.xctestplan

```diff
@@ -0,0 +1,75 @@
+{
+  "configurations" : [
+    {
+      "id" : "47D2FCAD-AC90-4E1C-BC4B-2B62F2EEC5E5",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "codeCoverage" : false
+  },
+  "testTargets" : [
+    {
+      "target" : {
+        "containerPath" : "container:Client.xcodeproj",
+        "identifier" : "2FA436041ABB83B4008031D1",
+        "name" : "AccountTests"
+      }
+    },
+    {
+      "skippedTests" : [
+        "ETPCoverSheetTests",
+        "TabManagerTests\/testDeleteSelectedTab()",
```

Comment:
I had to disable two unit tests in TabManager Test suite, four in MetaGlobalTests and one in StateTests. These are failing 100% when running the tests this way, no idea why... I have seen the MetaGlobal tests failing locally for me too, but they don't fail in regular builds... 
I will open issues for them so that we can re-enable asap

Reply:
> The Tab manager issue is this one correct? #10902

Yes, that is the issue
 
>  Do we have other issues for the MetaGlobalTests and StateTests ?
Not yet, because those are passing corrently with the current configuration. I will open them to remember that they are disabled an to try to find a fix...I will cc you there

Reply:
Ok thank you!

Reply:
Opened a [PR](https://github.com/mozilla-mobile/firefox-ios/pull/10989/files) to try to improve the TabManagerTests